README.md

```diff
@@ -7,7 +7,7 @@ distributed under a BSD/MIT license.
 ### Install ###
 At a minimum you will need to install:
 
-* Python 3 https://www.python.org/download/
+* Python 2 **and** 3 https://www.python.org/download/

```

Comment:
I don't think we need to explicitly mention Python2 here since it's only required if you follow the tutorial. So I think it suffices if we mention it there.